src/core/CMakeLists.txt

```diff
@@ -1269,8 +1269,13 @@ ADD_DEFINITIONS(-DTEST_DATA_DIR="${TEST_DATA_DIR}")
 
 #############################################################
 # qgis_core library
+IF (IOS)
```

Comment:
this should be move to top CMakeLists.txt so it will be available everywhere.
This could also be an advanced settings in CMake config?